test/performance/shmem_perf_suite/common.h

```diff
@@ -679,32 +669,30 @@ int error_checking_init_target_usage(perf_metrics_t *metric_info) {
     int error = false;
     assert(metric_info->midpt > 0);
 
-    if(metric_info->sztarget != -1 && metric_info->szinitiator != -1)
-        error = true; /* can't use them together  */
-
-    if(metric_info->sztarget != -1) {
-        if(metric_info->sztarget < 1 ||
-           metric_info->sztarget > metric_info->midpt ||
-           !metric_info->target_data) {
+    if (metric_info->sztarget != -1 && metric_info->szinitiator != -1) { 
```

Comment:
this if is empty

Reply:
Yes, kept it intentionally. Earlier, it was an error condition, now both the l and r flags can be used at the same time if user wants. If that looks fine, we will remove this if block.